docs/cpp/configure-intellisense.md

```diff
@@ -4,63 +4,67 @@ Area: cpp
+### Option 1. Select a configuration option through the configuration Quick Pick
 
-Open the quick pick by entering **Select IntelliSense Configuration** in the Command Palette (`kb(workbench.action.showCommands)`), which shows you a dropdown with all of the configuration options found by the C/C++ Extension on your machine.
+Open the Quick Pick by entering **Select IntelliSense Configuration** in the Command Palette (`kb(workbench.action.showCommands)`), which shows you a dropdown with all of the configuration options found by the C/C++ extension on your machine.
 
-![Compiler Quick Pick](images/intellisense/compiler-quick-pick.png)
+![Select a compiler to configure for IntelliSense Quick Pick](images/intellisense/compiler-quick-pick.png)
 
-Select one of the options available. If you select a compiler, this compiler is used by IntelliSense by default. You can return to the Configuration quick pick at any point to change which option is used to configure IntelliSense.
+Select one of the options available. If you select a compiler, this compiler is used by IntelliSense by default. You can return to the **Configure IntelliSense** Quick Pick at any point to change which option is used to configure IntelliSense.
 
-If no options are available in the quick pick, no compiler could be identified in your system. You can browse your machine manually or install a C/C++ compiler. To install on a Windows machine, select the **Help me install a compiler** option, which redirects you to the step-by-step walkthrough of how to install a C/C++ compiler. On a Mac or Linux machine, select **Install a compiler** and navigate through the prompts to have a C++ compiler installed on your machine.
+If no options are available in the Quick Pick, no compiler could be identified in your system. You can browse your machine manually or install a C/C++ compiler. To install on a Windows machine, select the **Help me install a compiler** option that redirects you to the step-by-step walkthrough of how to install a C/C++ compiler. On a macOS or Linux machine, select **Install a compiler** and navigate through the prompts to have a C++ compiler installed on your machine.
 
 ### Option 2. Edit your IntelliSense configurations through the UI
 
-Open your IntelliSense configuration by selecting **C/C++: Edit Configurations (UI)** from the Command Palette (`kb(workbench.action.showCommands)`). This view is an interface of the `c_cpp_properties.json` file.
+Open your IntelliSense configuration by selecting **C/C++: Edit Configurations (UI)** from the Command Palette (`kb(workbench.action.showCommands)`). This view is a user interface for the `c_cpp_properties.json` file.
 
 Set the **Compiler path** field to the full path of the compiler you're using to build your project. For example, when using the default install path for GCC on Linux, the compiler path is `/usr/bin/gcc`. Set the **IntelliSense mode** to the architecture-specific variant of the compiler you're using.
 
```

Comment:
There should also be a screenshot here: 
![image](https://github.com/microsoft/vscode-docs/assets/67650598/34e5ec73-fb19-41fd-9adc-0c075a452ef3)